research/src/pages/contribute.mdx

```diff
@@ -222,16 +263,28 @@ to be filed again the WHATWG HTML specification.
 
 **Note:** In order to understand how Open UI achieves consensus, please review to our [charter](https://open-ui.org/charter)
 
-## Denoting Questions
+### Denoting Questions
 
 There is an expectation that your proposal will have open questions. Leverage the following HTML to
 denote them, they'll be incremented automatically and pre-pended with 'Question'.
 
 `<p class="question">QUESTION</p>`
 
-## Try it out
+# Definitions <a href="#definitions" id="definitions"></a>
 
-You should now be able to `yarn start` the docs and see your component in the menu.
-You should also have a toggle on the component page to switch between the proposal page and research page.
+Since Open UI is about doing research across design systems to understand their components
+and controls it's important to have clear definitions. For the purposes of Open UI here are
+some common terms used in specifications and their definitions.
 
-:smile: You're all set! We're looking forward to your contribution to Open UI!
+## Component <a href="#definition-component" id="definition-component"></a>
+
+The dictionary defines a component as, "a part or element of a larger whole" and this is
+true for components in a design system's component library. Components normally have a meaning,
+although this meaning may not be presented to the user (such as a utility component). A component,
+as the definition states can be a part of another component which are known as **composite components**.
+
+## Control <a href="#definition-control" id="definition-control"></a>
+
+A control is a subset of component that allows some form of user interaction. Controls have a defined model,
```

Comment:
Do we want things like `<meter>` and `<progress>` to fall under the purview of this project?  Since the user can't manipulate them I'm not sure that they fit here unless we count reading the value to be "some form of user interaction"...but by that interpretation a `<div>` would match too because a user can read the contents.

That said I don't have a better suggestion at the moment, and it's probably not critical to bikeshed this part of the definition right now since IMO the more important part is the stuff about the defined model, parts, slots, states, and controller code.

Reply:
I did think about those, such as output or details/summary and I find those to be a component. So maybe the model and state should be moved up to component and that user interaction bit can stay on control?

Reply:
Hmm, yeah I think that would be a good way to break it down.

Reply:
Alrighty - I took a stab at adjusting it a bit. Give it another glance and let me know your thoughts.